src/api/TokenIO.m

```diff
@@ -62,7 +63,24 @@ - (id)initWithHost:(NSString *)host
         if (!useSsl) {
             [GRPCCall useInsecureConnectionsForHost:address];
         }
-        [GRPCCall setUserAgentPrefix:@"Token-iOS/1.0" forHost:address];
+        if (certsPath) {
+            NSError *error = nil;
+            NSString *certs = [NSString stringWithContentsOfFile:certsPath
+                                                        encoding:NSUTF8StringEncoding
+                                                           error:&error];
+            if (error) {
+                @throw [NSException exceptionWithName:@"InvalidCertificateException"
+                                               reason:@"The certificate data is invalid."
```

Comment:
if the error says "certificate data is invalid", I'm going to look at my file and try to figure out where I got the syntax wrong. but I ?think? this error actually means that I got the path wrong or else my file wasn't , uhm, text-y. I think this error message should change.

Reply:
change to throw the origin error instead of an encapsulated one